### DISCUSSION TITLE
Welcome to The HustleGPT Challenge!

### DISCUSSION THREAD

**Discussion by jtmuller5** [jtmuller5/The-HustleGPT-Challenge#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FilibusP** [comment]
Hi, I'm Promise, I want to join the HustleGPT community so I will learn how to create a side hustle, also to connect with members of the community because I love this project

**Comment by modorcelin** [comment]
Self taught programmer. Not that good but I'm hoping as I join more communities that will change. I hope I can contribute in one way or another. I would like add to the community/project or show casing what we are capable of as a collective

**Reply by jtmuller5** [reply]
Awesome! Glad to have you onboard. What programming language do you know?

**Reply by modorcelin** [reply]
Python

**Reply by jtmuller5** [reply]
Its a good one to know, especially with the AI hype these days. Looking forward to seeing what you build!

**Comment by thisolivier** [comment]
Hi there, I'm a cloud solutions architect, and front end iOS SDK specialist and an ex-graphic designer. I'm curious and excited about the potential of Hustle GPT. I'm interested in getting involved.
